homeassistant/components/soundtouch/strings.json

```diff
@@ -0,0 +1,24 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Setup a new Bose SoundTouch device",
+        "description": "Enter the device hostname or IP address of the Bose SoundTouch you want to add. Optionally, you can also specify the name to use.",
```

Comment:
Title is automatically set to manifest title if omitted, so not needed here.
Description also doesn't add much info. So let's drop it.
```suggestion
```